### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by Woodehh**
Readme update
 - Deprecated Google Docs spreadsheet
 - Added license
 - Added author
 - Added Adding yourself
 - Added city of zion logo
 - h1 fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,22 +66,8 @@ just run `gulp build`. It will now export a build to the public folder.
...
-
-
-Retrieve new team data:
-
-```bash
-gulp retrieveData
-```
+- Open-source [MIT](https://github.com/CityOfZion/neon-js/blob/master/LICENSE.md).
+- Main author is [Benjamin de Bos](https://github.com/woodehh).
+- Maintainer is [Chris Hager](https://github.com/metachris)
```
Comment (metachris) [inline README.md]: we should just add contributors: ... (hopefully add more soon)

File: README.md [hunk 2]
```diff
@@ -1,9 +1,40 @@
...
+<p align="center">
+	<a href="https://travis-ci.org/CityOfZion/cityofzion-website">
+		<img src="https://travis-ci.org/CityOfZion/cityofzion-website.svg?branch=master"/>
+	</a>
+</p>
+
+## Overview
+Very simple, no JS necessary. Using [Hugo](https://gohugo.io/), a fast static website generator and [NodeJS](https://nodejs.org) for building.
+
+## Adding yourself to the team
```
Comment (metachris) [inline README.md]: i don't think this should be at the top of the file. perhaps better after the run instructions

File: README.md [hunk 3]
```diff
@@ -1,9 +1,40 @@
...
+ - Council
+ - Advisor
+ - Maintainer
+ - Developer
+ - Contributor
+
+If you're NOT on the [City of Zion Github](https://github.com/orgs/CityOfZion/people), you're a contributor. If you're on there you're a developer. For the other roles: You know when you're a maintainer or advisor ;-)
+
+### Determined your role?
+ - Fork the repo
```
Comment (metachris) [inline README.md]: just link to the issue? it also has image optimization instructions

File: README.md (hunk 2 above)
Comment (metachris) [reply README.md]: nah i meant that how-to add yourself the to team instructions should go below the "how to run the dev server"

Comment (Woodehh) [reply README.md]: It's changed now

**Pull request merged**